script.js

```diff
@@ -0,0 +1,114 @@
+const colors = [
+    "#0fbcf9",
+    "#00d8d6",
+    "#05c46b",
+    "#ffc048",
+    "#ffdd59"
+  
+    ];
+
+  
+    let firstColor = colors[Math.floor(Math.random() * colors.length)];
+    let secondColor = colors[Math.floor(Math.random() * colors.length)];
+    
+    document.body.style.background = `linear-gradient(${firstColor}, ${secondColor})`;
+   
+    const toDoButton = document.getElementById("toDoButton");
+    const toDo = document.getElementById("toDo");
+    const toDoList = document.getElementById("todo-list");
+    const toDoInfo = document.getElementById("toDoInfo");
+    const doneList = document.getElementById("done-list");
+    const doneInfo = document.getElementById("doneInfo");
+    const finishtoDo = document.getElementById("finishtoDo");
+    
+    var toDoNum = 0;
+    var doneNum = 0;
```

Comment:
```suggestion
    let toDoNum = 0;
    let doneNum = 0;
```
ES6의 등장 이후로 `var`는 써야할 필요도, 써서도 안되는 예약어입니다! 거의 모든 상황에서 `var`는 `let/const`로 대체가 가능합니다. `var`가 가지는 특징들(특히 함수 범위 스코프 라던지)이 다른 프로그래밍 언어의 변수와는 조금 다르기 때문에 정확하게 알고 사용하지 않으면 예상 못한 버그를 초래할 수 있기 때문이에요.

Reply:
!!! 앞으로 var은 지양하겠습니다 ....*.*  그것도 모르고 var만 썼네요 ,,